### PR TITLE
Update fn_initPlayer.sqf

### LINKED ISSUE #11: If TIcketcost == 0, pls no notification thx

### PR DESCRIPTION
**Pull request by phumberdroz**
fix #11

### CODE REVIEW

**Comment by phumberdroz** [comment]
not sure if a 100% fix but I think it should work.

**Comment by Desty84** [comment]
Just from the function I would say, it will just display it as TicketGain ? :D

**Comment by DragonGER** [comment]
nope this will only change the message to be a ticket change instead ticket lost. 
What we need here is a new CBA setting option to disable notification on zero tickets.

`
if(_amount == 0 && { GVAR(enableZeroTicketNotification) }} exitWith {};
`

File: functions/ticket/fn_initPlayer.sqf [hunk 1]
```diff
@@ -21,10 +21,10 @@ call FUNC(addTicketDisplay);
 [
 	QGVAR(ticketChange), {
 		params ["_oldTickets", "_tickets", "_amount", "_message"];
-		if(_amount < 0) then {
+		if(_amount < -1) then {
```
Comment (phumberdroz) [inline functions/ticket/fn_initPlayer.sqf:24]:
```suggestion
		if(_amount < 0) then {
```

File: functions/ticket/fn_initPlayer.sqf [hunk 2]
```diff
@@ -21,10 +21,10 @@ call FUNC(addTicketDisplay);
 [
 	QGVAR(ticketChange), {
 		params ["_oldTickets", "_tickets", "_amount", "_message"];
```
Comment (phumberdroz) [inline functions/ticket/fn_initPlayer.sqf:23]:
```suggestion
		params ["_oldTickets", "_tickets", "_amount", "_message"];
		if(_amount == 0 && { GVAR(enableZeroTicketNotification) }} exitWith {};
```

**Comment by phumberdroz** [comment]
so like that? 👍

File: functions/ticket/fn_initPlayer.sqf [hunk 3]
```diff
@@ -21,10 +21,11 @@ call FUNC(addTicketDisplay);
...
-		if(_amount < 0) then {
+		if(_amount == 0 && { GVAR(enableZeroTicketNotification) }}) exitWith {};
```
Comment (DragonGER) [inline functions/ticket/fn_initPlayer.sqf]: damn... i added the ")" but that one "}" is still too much. sooorry

**Pull request merged**